unified-runtime/source/ur/ur.hpp

```diff
@@ -58,6 +58,8 @@ const ur_command_t UR_EXT_COMMAND_TYPE_USER =
 #define __SYCL_UR_PROGRAM_METADATA_TAG_MAX_LINEAR_WORK_GROUP_SIZE              \
   "@max_linear_work_group_size"
 #define __SYCL_UR_PROGRAM_METADATA_TAG_NEED_FINALIZATION "Requires finalization"
+#define __SYCL_UR_PROGRAM_METADATA_TAG_REQD_SUB_GROUP_SIZE                     \
```

Comment:
nit but I wonder if with respect to `__SYCL_UR_PROGRAM_METADATA_TAG_NEED_FINALIZATION` above it's better to keep all the `@` metadata items grouped together.

In fact, the comment says `Program metadata tags recognized by the UR adapters` and yet we have `Requires finalization` here, without a `@`? Is it in the wrong place?